libnmstate/nm/applier.py

```diff
@@ -401,7 +401,10 @@ def _build_connection_profile(
         bridge_options = iface_desired_state.get(bridge.BRIDGE_TYPE, {}).get(
             LB.OPTIONS_SUBTREE
         )
-        if bridge_options:
+        bridge_ports = iface_desired_state.get(bridge.BRIDGE_TYPE, {}).get(
```

Comment:
nit: It should be enough to do the `iface_desired_state.get(bridge.BRIDGE_TYPE, {})` once.